# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/transform-3d/reftest.list
# Commit: 5ccaf8100f79
# Full Hash: 5ccaf8100f7912e55bb98cc1951d57df9c4dd511
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-26 16:04:35
# Regressor Bug: 1766017
# File Overlap Count: 1
# Description:
#   Bug 1766017 - Do a better job of clipping off-screen surface allocation sizes r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D144534
# ==============================================================================

diff -r 01d5f09a02bf -r 5ccaf8100f79 layout/reftests/transform-3d/reftest.list
--- a/layout/reftests/transform-3d/reftest.list	Tue Apr 26 09:24:16 2022 +0000
+++ b/layout/reftests/transform-3d/reftest.list	Tue Apr 26 09:42:22 2022 +0000
@@ -94,7 +94,7 @@
 == intermediate-1.html intermediate-1-ref.html
 == preserves3d-nested-filter-1.html preserves3d-nested-filter-1-ref.html
 != preserve3d-scale.html about:blank
-fuzzy(0-50,0-1600) == preserve3d-scale.html preserve3d-scale-ref.html
+fuzzy(0-52,0-1600) == preserve3d-scale.html preserve3d-scale-ref.html
 fuzzy(0-1,0-5) == perspective-overflow-1.html perspective-overflow-1-ref.html
 == perspective-overflow-2.html perspective-overflow-2-ref.html
 == 1544995-1.html 1544995-1-ref.html
